<a href="https://colab.research.google.com/github/kittikhun62/DWDM21/blob/main/MiniExam.ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a>

### ข้อสอบ

จาก data ```flights.csv``` และ ```airports.csv``` ใน https://drive.google.com/drive/folders/1EFo1_uQgWkRiXjezL0T4YwteYrsqU5MV?usp=sharing
0. หารัฐที่มีจำนวนเครื่องบินขาเข้ามากที่สุดและน้อยที่สุด
1. หารัฐที่มีจำนวนเครื่องบินขาออกมากที่สุดและน้อยที่สุด
2. วาด Histogram แสดงจำนวนเครื่องบินขาออกของแต่ละวันในสัปดาห์
3. สายการบินไหนมีเที่ยวบินมากที่สุดและน้อยที่สุด
4. สายการบินอะไรมีความล่าช้ามากที่สุดและน้อยที่สุด
5. วันธรรมดาหรือเสาอาทิตย์ เครื่องบินขาเข้าจำนวนมากกว่ากัน(วันจันทร์ 1....7)

ให้นักศึกษาทำข้อสอบ 2 ข้อจากที่แสดงข้างบนโดยกรอกรหัสนักศึกษาลงใน function ```gen_exams()``` แล้ว run เพื่อให้ได้ข้อสอบที่นักศึกษาต้องทำ 2 ข้อ (ข้อละ 3 คะแนน)
และให้นักศึกษาคิดโจทย์เองอีก 1 ข้อ (4 คะแนน)


In [1]:
import pandas as pd

In [2]:
from google.colab import drive 
drive.mount('/content/drive')

Mounted at /content/drive


In [3]:
import os

In [4]:
flights = pd.read_csv('/content/drive/MyDrive/DWDM2021_DATA/Mini Exam/flights.csv')
flights

วันในเดือน  วันในสัปดาห์  ... ความล่าช้าขาออก  ความล่าช้าขาเข้า
0                19             5  ...              -3                 1
1                19             5  ...               0                -8
2                19             5  ...              -4               -15
3                19             5  ...              28                24
4                19             5  ...              -6               -11
...             ...           ...  ...             ...               ...
2702213          19             6  ...               1                -3
2702214          19             6  ...              -2                -3
2702215          19             6  ...              18                 9
2702216          19             6  ...              10                 7
2702217          19             6  ...               0               -13

[2702218 rows x 7 columns]

In [5]:
airports = pd.read_csv('/content/drive/MyDrive/DWDM2021_DATA/Mini Exam/airports.csv')
airports

รหัสสนามบิน         เมือง รัฐ                          ชื่อสนามบิน
0          10165   Adak Island  AK                                 Adak
1          10299     Anchorage  AK  Ted Stevens Anchorage International
2          10304         Aniak  AK                        Aniak Airport
3          10754        Barrow  AK      Wiley Post/Will Rogers Memorial
4          10551        Bethel  AK                       Bethel Airport
..           ...           ...  ..                                  ...
360        11233      Cheyenne  WY  Cheyenne Regional/Jerry Olson Field
361        11097          Cody  WY                 Yellowstone Regional
362        11865      Gillette  WY             Gillette Campbell County
363        12441       Jackson  WY                         Jackson Hole
364        14543  Rock Springs  WY       Rock Springs Sweetwater County

[365 rows x 4 columns]

In [6]:
def gen_exams(ID):
    import math
    a = int(ID[-1])%6
    b = int(ID[-1])%3
    print(f'รหัสนักศึกษา {ID} ทำข้อ {a} และ {b if a != b else 5-b}')

ตัวอย่างการ run

In [7]:
gen_exams('623020760-0')

รหัสนักศึกษา 623020760-0 ทำข้อ 0 และ 5


ข้อ 0 หารัฐที่มีจำนวนเครื่องบินขาเข้ามากที่สุดและน้อยที่สุด

In [8]:
merged_table = airports.merge(flights,how='left',left_on='รหัสสนามบิน',right_on='รหัสสนามบินปลายทาง')
merged_table #(เชื่อมตารางโดยใช้ index ของ รหัสสนามบินเเละรหัสสนามบินต้นทาง)

รหัสสนามบิน         เมือง  ... ความล่าช้าขาออก ความล่าช้าขาเข้า
0              10165   Adak Island  ...             NaN              NaN
1              10299     Anchorage  ...            21.0             23.0
2              10299     Anchorage  ...            -1.0              7.0
3              10299     Anchorage  ...            -4.0             -4.0
4              10299     Anchorage  ...            13.0             10.0
...              ...           ...  ...             ...              ...
2702508        11233      Cheyenne  ...             NaN              NaN
2702509        11097          Cody  ...             NaN              NaN
2702510        11865      Gillette  ...             NaN              NaN
2702511        12441       Jackson  ...             NaN              NaN
2702512        14543  Rock Springs  ...             NaN              NaN

[2702513 rows x 11 columns]

### ทำการลบ missing โดยใช้ Dropna

In [9]:
merged_table.isnull().any()

รหัสสนามบิน           False
เมือง                 False
รัฐ                   False
ชื่อสนามบิน           False
วันในเดือน             True
วันในสัปดาห์           True
สายการบิน              True
รหัสสนามบินต้นทาง      True
รหัสสนามบินปลายทาง     True
ความล่าช้าขาออก        True
ความล่าช้าขาเข้า       True
dtype: bool

In [23]:
merged_table_dropna = merged_table.dropna()
merged_table_dropna 

รหัสสนามบิน      เมือง  ... ความล่าช้าขาออก ความล่าช้าขาเข้า
1              10299  Anchorage  ...            21.0             23.0
2              10299  Anchorage  ...            -1.0              7.0
3              10299  Anchorage  ...            -4.0             -4.0
4              10299  Anchorage  ...            13.0             10.0
5              10299  Anchorage  ...            24.0             39.0
...              ...        ...  ...             ...              ...
2702496        13342  Milwaukee  ...            20.0              5.0
2702497        13342  Milwaukee  ...            11.0             17.0
2702498        13342  Milwaukee  ...            -2.0             -6.0
2702499        13342  Milwaukee  ...            -5.0              0.0
2702500        13342  Milwaukee  ...            -1.0            -12.0

[2702218 rows x 11 columns]

In [13]:
merged_table_dropna.shape #จะเห็นได้ว่าหลังจากdropna มีข้อมูลของเเถวหายไปเป็นเเถวที่มีช่องว่าง Nan

(2702218, 11)

 ขาเข้ามากสุด เเละ ขาเข้าน้อยสุด


In [16]:
total = merged_table_dropna.groupby('รัฐ')['ความล่าช้าขาเข้า'].max()
total = pd.DataFrame(total)
total.reset_index()  #นำตารางที่เป็น dropna มาเลือกเเค่คอลัมที่เราต้องการ

รัฐ  ความล่าช้าขาเข้า
0   AK             623.0
1   AZ            1099.0
2   CA            1228.0
3   CO             776.0
4   CT             448.0
5   DC             637.0
6   FL            1475.0
7   GA            1146.0
8   HI            1845.0
9   IL            1835.0
10  IN             427.0
11  KY             452.0
12  LA             788.0
13  MA             933.0
14  MD             795.0
15  MI             988.0
16  MN            1116.0
17  MO             802.0
18  NC             597.0
19  NE             413.0
20  NJ             645.0
21  NM             424.0
22  NV             852.0
23  NY            1179.0
24  OH            1143.0
25  OK             560.0
26  OR             850.0
27  PA             848.0
28  PR             407.0
29  RI             415.0
30  TN             614.0
31  TX            1609.0
32  UT            1120.0
33  VA             995.0
34  WA             975.0
35  WI             811.0

In [72]:
max(total['ความล่าช้าขาเข้า']) # หาค่ามากสุดของตาราง

1845.0

In [73]:
total.describe() #ดูค่าทางสถิติเพื่อความเเน่ใจ

ความล่าช้าขาเข้า
count         36.000000
mean         885.277778
std          385.106487
min          407.000000
25%          609.750000
50%          829.500000
75%         1117.000000
max         1845.000000

In [74]:
total[total['ความล่าช้าขาเข้า'] == 1845]  # ดั้งนั้นรัฐที่มีจำนวนเที่ยวบินขาเข้ามากที่สุดคือ รัฐ HI

ความล่าช้าขาเข้า
รัฐ                  
HI             1845.0

In [75]:
total_min = merged_table_dropna.groupby('รัฐ')['ความล่าช้าขาเข้า'].min()
total_min = pd.DataFrame(total_min)
total_min.reset_index()  #นำตารางที่เป็น dropna มาเลือกเเค่คอลัมที่เราต้องการ เพื่อหาค่าน้อยสุดของขาเข้า

รัฐ  ความล่าช้าขาเข้า
0   AK             -60.0
1   AZ             -71.0
2   CA             -86.0
3   CO             -66.0
4   CT             -43.0
5   DC             -94.0
6   FL             -63.0
7   GA             -52.0
8   HI             -64.0
9   IL             -62.0
10  IN             -52.0
11  KY             -52.0
12  LA             -65.0
13  MA             -60.0
14  MD             -53.0
15  MI             -51.0
16  MN             -68.0
17  MO             -68.0
18  NC             -53.0
19  NE             -45.0
20  NJ             -67.0
21  NM             -58.0
22  NV             -69.0
23  NY             -69.0
24  OH             -57.0
25  OK             -53.0
26  OR             -71.0
27  PA             -57.0
28  PR             -56.0
29  RI             -36.0
30  TN             -62.0
31  TX             -68.0
32  UT             -63.0
33  VA             -61.0
34  WA             -75.0
35  WI             -63.0

In [76]:
total_min.describe() #ดูค่าทางสถิติ

ความล่าช้าขาเข้า
count         36.000000
mean         -61.472222
std           11.064709
min          -94.000000
25%          -68.000000
50%          -62.000000
75%          -53.000000
max          -36.000000

In [77]:
min(total_min['ความล่าช้าขาเข้า']) #หาค่าที่น้อยที่สุดโดยใช้ค่า min

-94.0

In [78]:
total_min[total_min['ความล่าช้าขาเข้า'] == -94.0] #ดึงข้อมูลให้ออกมาเป็นค่าในตารางด้วย

ความล่าช้าขาเข้า
รัฐ                  
DC              -94.0

italicised text ตอบข้อ 0 รัฐที่มีจำนวนเครื่องบินขาเข้ามากที่สุดคือรัฐ HI 1845 รำ เเละ น้อยสุดคือรัฐ DC -94 รำ

## 5. วันธรรมดาหรือเสาอาทิตย์ เครื่องบินขาเข้าจำนวนมากกว่ากัน(วันจันทร์ 1....7) โผมคิดไม่ออกครับTT

รหัสสนามบิน      เมือง  ... ความล่าช้าขาออก ความล่าช้าขาเข้า
1              10299  Anchorage  ...            21.0             23.0
2              10299  Anchorage  ...            -1.0              7.0
3              10299  Anchorage  ...            -4.0             -4.0
4              10299  Anchorage  ...            13.0             10.0
5              10299  Anchorage  ...            24.0             39.0
...              ...        ...  ...             ...              ...
2702496        13342  Milwaukee  ...            20.0              5.0
2702497        13342  Milwaukee  ...            11.0             17.0
2702498        13342  Milwaukee  ...            -2.0             -6.0
2702499        13342  Milwaukee  ...            -5.0              0.0
2702500        13342  Milwaukee  ...            -1.0            -12.0

[2702218 rows x 11 columns]

In [39]:
merged_table_dropna[merged_table_dropna['วันในสัปดาห์'] == 1]

รหัสสนามบิน      เมือง  ... ความล่าช้าขาออก ความล่าช้าขาเข้า
12             10299  Anchorage  ...            -1.0             11.0
13             10299  Anchorage  ...            14.0             46.0
19             10299  Anchorage  ...             0.0             40.0
35             10299  Anchorage  ...            -6.0             15.0
36             10299  Anchorage  ...            -1.0             -8.0
...              ...        ...  ...             ...              ...
2702457        13342  Milwaukee  ...            -1.0             -7.0
2702458        13342  Milwaukee  ...            -4.0             -7.0
2702459        13342  Milwaukee  ...            -3.0            -16.0
2702460        13342  Milwaukee  ...            27.0             11.0
2702461        13342  Milwaukee  ...             0.0             -2.0

[407837 rows x 11 columns]

In [66]:
merged_table_dropna[merged_table_dropna['วันในสัปดาห์'] == 2]

รหัสสนามบิน      เมือง  ... ความล่าช้าขาออก ความล่าช้าขาเข้า
3              10299  Anchorage  ...            -4.0             -4.0
4              10299  Anchorage  ...            13.0             10.0
16             10299  Anchorage  ...             0.0              5.0
17             10299  Anchorage  ...             2.0              7.0
22             10299  Anchorage  ...             2.0              7.0
...              ...        ...  ...             ...              ...
2702466        13342  Milwaukee  ...            -2.0             -8.0
2702467        13342  Milwaukee  ...            12.0              8.0
2702480        13342  Milwaukee  ...            -2.0            -14.0
2702481        13342  Milwaukee  ...            -7.0             23.0
2702482        13342  Milwaukee  ...            -1.0              5.0

[397594 rows x 11 columns]

In [67]:
merged_table_dropna[merged_table_dropna['วันในสัปดาห์'] == 3]

รหัสสนามบิน      เมือง  ... ความล่าช้าขาออก ความล่าช้าขาเข้า
5              10299  Anchorage  ...            24.0             39.0
18             10299  Anchorage  ...            12.0             14.0
20             10299  Anchorage  ...            -5.0             27.0
21             10299  Anchorage  ...             0.0             11.0
25             10299  Anchorage  ...            14.0             33.0
...              ...        ...  ...             ...              ...
2702475        13342  Milwaukee  ...            -5.0              3.0
2702476        13342  Milwaukee  ...            -4.0             -7.0
2702477        13342  Milwaukee  ...            -2.0            -10.0
2702483        13342  Milwaukee  ...            -2.0            -11.0
2702484        13342  Milwaukee  ...             2.0             -9.0

[403072 rows x 11 columns]

In [68]:
merged_table_dropna[merged_table_dropna['วันในสัปดาห์'] == 4]

รหัสสนามบิน      เมือง  ... ความล่าช้าขาออก ความล่าช้าขาเข้า
1              10299  Anchorage  ...            21.0             23.0
6              10299  Anchorage  ...             5.0             57.0
7              10299  Anchorage  ...            56.0             92.0
23             10299  Anchorage  ...            -4.0             24.0
24             10299  Anchorage  ...             2.0             12.0
...              ...        ...  ...             ...              ...
2702494        13342  Milwaukee  ...            -4.0            -12.0
2702495        13342  Milwaukee  ...            16.0             17.0
2702496        13342  Milwaukee  ...            20.0              5.0
2702497        13342  Milwaukee  ...            11.0             17.0
2702498        13342  Milwaukee  ...            -2.0             -6.0

[406563 rows x 11 columns]

In [69]:
merged_table_dropna[merged_table_dropna['วันในสัปดาห์'] == 5]

รหัสสนามบิน      เมือง  ... ความล่าช้าขาออก ความล่าช้าขาเข้า
2              10299  Anchorage  ...            -1.0              7.0
27             10299  Anchorage  ...            -3.0              8.0
28             10299  Anchorage  ...            -3.0             -2.0
30             10299  Anchorage  ...            -2.0              2.0
54             10299  Anchorage  ...            -2.0              5.0
...              ...        ...  ...             ...              ...
2702420        13342  Milwaukee  ...             4.0              4.0
2702421        13342  Milwaukee  ...            36.0             24.0
2702422        13342  Milwaukee  ...             9.0             -1.0
2702499        13342  Milwaukee  ...            -5.0              0.0
2702500        13342  Milwaukee  ...            -1.0            -12.0

[396387 rows x 11 columns]

In [70]:
merged_table_dropna[merged_table_dropna['วันในสัปดาห์'] == 6]

รหัสสนามบิน      เมือง  ... ความล่าช้าขาออก ความล่าช้าขาเข้า
8              10299  Anchorage  ...             0.0            -13.0
9              10299  Anchorage  ...             0.0              6.0
11             10299  Anchorage  ...            -7.0             12.0
29             10299  Anchorage  ...            -7.0             -4.0
31             10299  Anchorage  ...             4.0             -2.0
...              ...        ...  ...             ...              ...
2702424        13342  Milwaukee  ...            16.0             -3.0
2702425        13342  Milwaukee  ...            16.0              9.0
2702426        13342  Milwaukee  ...            -8.0            -11.0
2702427        13342  Milwaukee  ...            -6.0            -20.0
2702428        13342  Milwaukee  ...             0.0             -3.0

[318537 rows x 11 columns]

In [71]:
merged_table_dropna[merged_table_dropna['วันในสัปดาห์'] == 7]

รหัสสนามบิน      เมือง  ... ความล่าช้าขาออก ความล่าช้าขาเข้า
10             10299  Anchorage  ...            -1.0              9.0
14             10299  Anchorage  ...            -4.0            -10.0
15             10299  Anchorage  ...             3.0             10.0
33             10299  Anchorage  ...            -6.0              8.0
34             10299  Anchorage  ...            11.0             12.0
...              ...        ...  ...             ...              ...
2702444        13342  Milwaukee  ...            -4.0            -11.0
2702445        13342  Milwaukee  ...            -1.0             -9.0
2702446        13342  Milwaukee  ...            -6.0            -10.0
2702447        13342  Milwaukee  ...            -5.0            -11.0
2702448        13342  Milwaukee  ...            -5.0            -18.0

[372228 rows x 11 columns]

#โจทย์ของตัวเอง คือ ถ้าต้องการทราบข้อมูลว่า ในวันเสาร์ เครื่องบินขาออกไปรัฐอะไรเเละสนามบินไหนมากที่สุด วันเสาร์คือเลข 6

In [40]:
merged_table_dropna

รหัสสนามบิน      เมือง  ... ความล่าช้าขาออก ความล่าช้าขาเข้า
1              10299  Anchorage  ...            21.0             23.0
2              10299  Anchorage  ...            -1.0              7.0
3              10299  Anchorage  ...            -4.0             -4.0
4              10299  Anchorage  ...            13.0             10.0
5              10299  Anchorage  ...            24.0             39.0
...              ...        ...  ...             ...              ...
2702496        13342  Milwaukee  ...            20.0              5.0
2702497        13342  Milwaukee  ...            11.0             17.0
2702498        13342  Milwaukee  ...            -2.0             -6.0
2702499        13342  Milwaukee  ...            -5.0              0.0
2702500        13342  Milwaukee  ...            -1.0            -12.0

[2702218 rows x 11 columns]

In [79]:
data = merged_table_dropna[merged_table_dropna['วันในสัปดาห์'] == 6]
data #ให้เลือกตารางที่มีเเค่เลข 6 คือเเสดงข้อมูลทั้งหมดที่เป็นวันเสาร์

รหัสสนามบิน      เมือง  ... ความล่าช้าขาออก ความล่าช้าขาเข้า
8              10299  Anchorage  ...             0.0            -13.0
9              10299  Anchorage  ...             0.0              6.0
11             10299  Anchorage  ...            -7.0             12.0
29             10299  Anchorage  ...            -7.0             -4.0
31             10299  Anchorage  ...             4.0             -2.0
...              ...        ...  ...             ...              ...
2702424        13342  Milwaukee  ...            16.0             -3.0
2702425        13342  Milwaukee  ...            16.0              9.0
2702426        13342  Milwaukee  ...            -8.0            -11.0
2702427        13342  Milwaukee  ...            -6.0            -20.0
2702428        13342  Milwaukee  ...             0.0             -3.0

[318537 rows x 11 columns]

In [80]:
data_sat = data.groupby('รัฐ')['ความล่าช้าขาออก'].max()
data_sat = pd.DataFrame(data_sat)
data_sat.reset_index()  #ดึงตารางข้อมูลที่ต้องการออกมา

รัฐ  ความล่าช้าขาออก
0   AK            364.0
1   AZ            509.0
2   CA           1222.0
3   CO            726.0
4   CT            218.0
5   DC            536.0
6   FL            669.0
7   GA           1036.0
8   HI            881.0
9   IL           1380.0
10  IN            366.0
11  KY            306.0
12  LA            273.0
13  MA            440.0
14  MD            353.0
15  MI            524.0
16  MN            660.0
17  MO            769.0
18  NC            407.0
19  NE            270.0
20  NJ            589.0
21  NM            379.0
22  NV            494.0
23  NY            796.0
24  OH           1137.0
25  OK            339.0
26  OR            500.0
27  PA            663.0
28  PR            319.0
29  RI            327.0
30  TN            408.0
31  TX           1214.0
32  UT            732.0
33  VA            826.0
34  WA            522.0
35  WI            337.0

In [81]:
data_sat_1 = data_sat.reset_index()
data_sat_1 #นำค่าตารางไปเเทนในตัวเเปรเพื่อจะนำมาใช้เชื่อมตาราง

รัฐ  ความล่าช้าขาออก
0   AK            364.0
1   AZ            509.0
2   CA           1222.0
3   CO            726.0
4   CT            218.0
5   DC            536.0
6   FL            669.0
7   GA           1036.0
8   HI            881.0
9   IL           1380.0
10  IN            366.0
11  KY            306.0
12  LA            273.0
13  MA            440.0
14  MD            353.0
15  MI            524.0
16  MN            660.0
17  MO            769.0
18  NC            407.0
19  NE            270.0
20  NJ            589.0
21  NM            379.0
22  NV            494.0
23  NY            796.0
24  OH           1137.0
25  OK            339.0
26  OR            500.0
27  PA            663.0
28  PR            319.0
29  RI            327.0
30  TN            408.0
31  TX           1214.0
32  UT            732.0
33  VA            826.0
34  WA            522.0
35  WI            337.0

In [56]:
data_sat_1.describe()

ความล่าช้าขาออก
count        36.000000
mean        596.972222
std         302.259915
min         218.000000
25%         361.250000
50%         515.500000
75%         741.250000
max        1380.000000

In [82]:
max(data_sat_1['ความล่าช้าขาออก']) #หาค่าที่มีเครื่องบินค่าออกมากสุดของวันเสาร์

1380.0

In [83]:
data_sat_1[data_sat_1['ความล่าช้าขาออก'] == 1380.0] #เลือกค่าในตารางออกมา

รัฐ  ความล่าช้าขาออก
9  IL           1380.0

In [84]:
data_Location = data.groupby('ชื่อสนามบิน')['ความล่าช้าขาออก'].max()
data_Location = pd.DataFrame(data_Location)
data_Location.reset_index()  #ดึงตารางข้อมูลที่เป็นชื่อสนามบิน

ชื่อสนามบิน  ความล่าช้าขาออก
0                   Albuquerque International Sunport            379.0
1                    Austin - Bergstrom International            664.0
2   Baltimore/Washington International Thurgood Ma...            353.0
3                                            Bob Hope            199.0
4                               Bradley International            218.0
..                                                ...              ...
65                       Theodore Francis Green State            327.0
66                               Tucson International            198.0
67                    Washington Dulles International            536.0
68                                  Will Rogers World            339.0
69                                    William P Hobby            406.0

[70 rows x 2 columns]

In [85]:
data_LO = data_Location.reset_index() 
data_LO #เเทนตัวเเปรให้ในตาราง

ชื่อสนามบิน  ความล่าช้าขาออก
0                   Albuquerque International Sunport            379.0
1                    Austin - Bergstrom International            664.0
2   Baltimore/Washington International Thurgood Ma...            353.0
3                                            Bob Hope            199.0
4                               Bradley International            218.0
..                                                ...              ...
65                       Theodore Francis Green State            327.0
66                               Tucson International            198.0
67                    Washington Dulles International            536.0
68                                  Will Rogers World            339.0
69                                    William P Hobby            406.0

[70 rows x 2 columns]

In [59]:
max(data_LO['ความล่าช้าขาออก']) 

1380.0

In [60]:
data_LO[data_LO['ความล่าช้าขาออก'] == 1380.0] 

ชื่อสนามบิน  ความล่าช้าขาออก
8  Chicago O'Hare International           1380.0

In [86]:
data1 = data_sat_1[data_sat_1['ความล่าช้าขาออก'] == 1380.0] 
data2 = data_LO[data_LO['ความล่าช้าขาออก'] == 1380.0]  #เเทนดาต้าในตัวเเปรเพื่อใช้นำมาเป็นตัวเชื่อมข้อมูล

In [87]:
data_total_1 = data1.merge(data2,how='left',left_on='ความล่าช้าขาออก',right_on='ความล่าช้าขาออก')
data_total_1  #เชื่อมตารางโดยใช้ ความล่าช้าขาออก เป็น index ในการเชื่อม เพื่อเเสดงด้าต้าให้เป็นเเถวในเเกน X

รัฐ  ความล่าช้าขาออก                   ชื่อสนามบิน
0  IL           1380.0  Chicago O'Hare International

**ดังนั้นน ในวันเสาร์ เครื่องบินขาออกไปรัฐ IL เเละสนามบิน  Chicago O'Hare International ในวันเสาร์มากที่สุด อยู่ที่ 1380 รำ**